package.json

```diff
@@ -34,8 +30,8 @@
     "streamify-array": "^1.0.0",
     "streamify-string": "^1.0.1",
     "ts-jest": "^29.0.0",
-    "typedoc": "^0.21.0",
-    "typescript": "^4.5.2",
+    "typedoc": "^0.25.1",
```

Comment:
This update is not so straightforward if I remember correctly, see https://github.com/comunica/comunica/commit/5a2dbe480ae83fc7de7769355abe6cefb4e65881

We'll also have to remove any job deps from the CI workflow file, so the docs job can start immediately (as it takes a long time)

Reply:
I have included the necessary changes to make TypeDoc work, but instead of two bash scripts I combined them into a single one. The docs CI job now also starts without any conditions.

Reply:
I would like these script so be identical across all comunica repos, so we can easily copy-paste upon changes.
Could you either split this file here up again, or make a copy of your file into comunica base?